File: 7kyu/Exes and Ohs/exesAndOhs.php
<?php
function XO($s)
{
    $array = str_split($s);
    $count = 0;

    foreach ($array as $item) {
        if (strtolower($item) == "x") {
            $count++;
        } elseif (strtolower($item) == "o") {
            $count--;
        }
    }


    return $count === 0;
}

File: 7kyu/Exes and Ohs/exesAndOhs.py
def xo(s):
    array = list(s)
    count = 0
    
    for item in array:
        if item.lower() == "x":
            count += 1
        elif  item.lower() == "o":
            count -= 1
            
            
    return count == 0




xo("hello mate")